pr2_plugs_configuration: 
  grasp_plug_seed:
    joint_names: ['r_shoulder_pan_joint', 'r_shoulder_lift_joint', 'r_upper_arm_roll_joint', 
                  'r_elbow_flex_joint', 'r_forearm_roll_joint', 'r_wrist_flex_joint', 'r_wrist_roll_joint']
    seed_position: 
    - [-0.42972163482294801, 0.99393222878020571, -2.9066834445806746, -1.1962312380847619, 
      1.0253396733773847, -0.99226493985362119, 2.6919730975699872]



  
